ecosystem/python/sdk/examples/rotate_key.py

```diff
@@ -122,13 +167,26 @@ async def main():
     ), "Authentication key doesn't match Bob's"
 
     # Construct a new Account object that reflects alice's original address with the new private key
+    original_alice_key = alice.private_key
     alice = Account(alice.address(), bob.private_key)
 
     # Display formatted account info
     print("Alice".ljust(WIDTH, " ") + format_account_info(alice))
     print("Bob".ljust(WIDTH, " ") + format_account_info(bob))
     print()
 
+    print("\n...rotating...\n")
+    payload = await rotate_auth_key_multi_ed_25519_payload(
+        rest_client, alice, [bob.private_key, original_alice_key]
+    )
+    signed_transaction = await rest_client.create_bcs_signed_transaction(alice, payload)
+    tx_hash = await rest_client.submit_bcs_transaction(signed_transaction)
+    await rest_client.wait_for_transaction(tx_hash)
+
+    alice_new_account_info = await rest_client.account(alice.address())
+    auth_key = alice_new_account_info["authentication_key"]
+    print(f"Rotation to MultiPublicKey complete, new authkey: {auth_key}")
+
     await rest_client.close()
 
```

Comment:
Can you add a test for using the multied25519?

Reply:
there are some in the code base and unfortunately the rest of the code isn't implemented... maybe I can slap another PR on top of this for richer functionality...

Reply:
gonna punt on this now... using multied is a pain because the account abstraction isn't great... I think once we improve the area around a Multi Account Authenticator TA we can revisit.